keyboards/mechlovin/infinityce/keymaps/via/keymap.c

```diff
@@ -0,0 +1,53 @@
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+        [0] = LAYOUT_all(
+        KC_ESC,     KC_F1,     KC_F2,     KC_F3,     KC_F4,     KC_F5,     KC_F6,     KC_F7,     KC_F8,     KC_F9,     KC_F10,   KC_F11,      KC_F12,    RGB_MOD,                 KC_PSCR,     KC_SLCK,     KC_PAUS,
+        KC_GRV,     KC_1,      KC_2,      KC_3,      KC_4,      KC_5,      KC_6,      KC_7,      KC_8,      KC_9,      KC_0,     KC_MINS,     RGB_HUD,   KC_BSPC,                 KC_INS,      KC_HOME,     KC_PGUP,
+        KC_TAB,     KC_Q,      KC_W,      RGB_MOD,   KC_R,      KC_T,      RGB_MOD,   KC_U,      KC_I,      KC_O,      KC_P,     KC_LBRC,     KC_RBRC,   KC_SLSH,                 KC_DEL,      KC_END,      KC_PGDN,
+        KC_CAPS,    KC_A,      KC_S,      KC_D,      KC_F,      KC_G,      KC_H,      KC_J,      KC_K,      KC_L,      KC_SCLN,  KC_QUOT,     KC_NUHS,   KC_ENT,
+        KC_LSFT,    KC_NUBS,   KC_Z,      KC_X,      KC_C,      KC_V,      KC_B,      KC_N,      KC_M,      KC_COMM,   KC_DOT,   KC_BSLS,     MO(1),     KC_LSFT,                              KC_UP,
+        KC_LCTL,    KC_LGUI,   KC_LALT,                                               KC_SPC,                                    KC_RALT,     KC_RGUI,   KC_APP,     KC_RCTL,     KC_LEFT,     KC_DOWN,     KC_RGHT
+    ),
+        [0] = LAYOUT_all(
+        KC_ESC,     KC_F1,     KC_F2,     KC_F3,     KC_F4,     KC_F5,     KC_F6,     KC_F7,     KC_F8,     KC_F9,     KC_F10,   KC_F11,      KC_F12,    RGB_MOD,                 KC_PSCR,     KC_SLCK,     KC_PAUS,
+        KC_GRV,     KC_1,      KC_2,      KC_3,      KC_4,      KC_5,      KC_6,      KC_7,      KC_8,      KC_9,      KC_0,     KC_MINS,     RGB_HUD,   KC_BSPC,                 KC_INS,      KC_HOME,     KC_PGUP,
+        KC_TAB,     KC_Q,      KC_W,      RGB_MOD,   KC_R,      KC_T,      RGB_MOD,   KC_U,      KC_I,      KC_O,      KC_P,     KC_LBRC,     KC_RBRC,   KC_SLSH,                 KC_DEL,      KC_END,      KC_PGDN,
+        KC_CAPS,    KC_A,      KC_S,      KC_D,      KC_F,      KC_G,      KC_H,      KC_J,      KC_K,      KC_L,      KC_SCLN,  KC_QUOT,     KC_NUHS,   KC_ENT,
+        KC_LSFT,    KC_NUBS,   KC_Z,      KC_X,      KC_C,      KC_V,      KC_B,      KC_N,      KC_M,      KC_COMM,   KC_DOT,   KC_BSLS,     MO(1),     KC_LSFT,                              KC_UP,
+        KC_LCTL,    KC_LGUI,   KC_LALT,                                               KC_SPC,                                    KC_RALT,     KC_RGUI,   KC_APP,     KC_RCTL,     KC_LEFT,     KC_DOWN,     KC_RGHT
+    ),
+        [0] = LAYOUT_all(
```

Comment:
```suggestion
        [2] = LAYOUT_all(
```

You also could and probably should fill this layer and the next with `KC_TRNS` (`_______`).